bridges/EdfColorDayBridge.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+class EdfColorDayBridge extends BridgeAbstract
+{
+    const NAME = 'EDF tempo color';
+    // pull info from this site for now because EDF do not provide correct opendata
+    const URI = 'https://www.services-rte.com/cms/open_data/v1/tempo';
+    const DESCRIPTION = 'Get EDF color of today and tomorrow of tempo contract';
+    const MAINTAINER = 'floviolleau';
+    const PARAMETERS = [
+        [
+            'contract' => [
+                'name' => 'Choisir un contrat',
+                'type' => 'list',
+                // we can add later more option prices like EJP
+                'values' => [
+                    'Tempo' => 'tempo'
+                ],
+            ]
+        ]
+    ];
+    const CACHE_TIMEOUT = 7200; // 2h
+
+    /**
+     * @param simple_html_dom $html
+     * @param string $contractUri
+     * @return void
+     */
+    private function tempo(string $json): void
+    {
+        $jsonDecoded = json_decode($json, true, 512, JSON_THROW_ON_ERROR);
+
+        $values = [
+            $this->formatFrenchDate('now') => date('Y-m-d'),
+            'Demain ' . $this->formatFrenchDate('tomorrow') => date('Y-m-d', strtotime('+1 day'))
+        ];
+
+        foreach ($values as $key => $value) {
+            $i++;
+            $item = [];
+
+            $text = $key . ' : ' . $this->getDisplayableColor($jsonDecoded['values'][$value]);
+            $item['uri'] = self::URI . $contractUri;
+            $item['title'] = $text;
+            $item['author'] = self::MAINTAINER;
+            $item['content'] = $text;
+            $item['uid'] = hash('sha256', $item['title']);
+
+            $this->items[] = $item;
+        }
+    }
+
+    private function formatFrenchDate(string $datetime): string
+    {
+        // Set the locale to French
+        setlocale(LC_TIME, 'fr_FR.UTF-8');
+
```

Comment:
Please do not change locale. Can you find another way?

Reply:
I did something different. Tell me if it is OK now.
This bridge is mainly for French people so I still set the date format to FR but if you don't feel comfortable with it I can add an input parameter of the bridge.